File: 26. Remove Duplicates from Sorted Array/Remove Duplicates from Sorted Array.cs
public class Solution 
{
    public int RemoveDuplicates(int[] nums) 
    {
        if(nums.Length == 0)
        {
            return 0;
        }
        
        int counter = 0;
        for(int j = 1; j < nums.Length; j++)
        {
            if(nums[counter] != nums[j])
            {
                counter++;
                nums[counter] = nums[j];
            }
        }
        
        return 1 + counter;
    }
}

File: 26. Remove Duplicates from Sorted Array/Remove Duplicates from Sorted Array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        if (nums.length == 0)
        {
            return 0;
        }
        
        int i = 0; //i -> counter
        
        for (int j = 1; j < nums.length; j++ )
        {
            if(nums[j] != nums[i])
            {
                i++;
                nums[i] = nums[j];
            }
            
        }
        return 1 + i;
    }
}